### DISCUSSION TITLE
Allow to add multiple spammer channel IDs at once

### DISCUSSION THREAD

**Discussion by SevastianGit** [ThioJoe/YT-Spammer-Purge#39] [Ideas]
This would help in case if your channel/video has many different spammers, you just copy all of their IDs, put them into the software and delete all of their comments at once without having to repeat the scanning and deletion for every channel. Although Google's quota might be an issue.

**Comment by http403** [comment]
possible dup of #18

**Comment by SevastianGit** [comment]
> possible dup of #18
That issue talks about a **list** that anyone could use, my issue talks about a feature that you can add multiple channel IDs at once that you collect yourself, kinda different.

**Comment by http403** [comment]
Not to argue but if you talk about a community run list, that would be #15. Otherwise, I can't really see a different. List or not, just a choice of implementation that can be discussed later.

**Comment by KendallDoesCoding** [comment]
Well, you can just open the app 2 times, While this idea is cool, it may take lots of work to be added, but good idea ig.

**Comment by KendallDoesCoding** [comment]


**Reply by ArjixWasTaken** [reply]
ok now that's some bullshit
that's only possible if you add A LOT and you are running the program using a micro system like a raspberry pi

**Reply by KendallDoesCoding** [reply]
What I meant to say is that it will take a long time to scan the youtube
videos and the comments and if you wanted you could just open the app 2
times to save time.....
…

**Comment by KendallDoesCoding** [comment]
Well, I think this may already be possible because in the logs it says channel ids of scammer and displays their one id, please confirm if it is a feature @ThioJoe

**Comment by ThioJoe** [comment]
Yea you can now search by putting in multiple comma-separated links / IDs. The latest python script already has it, but for the exe version it will be in the next release.

**Reply by KendallDoesCoding** [reply]
Ok cool. You can close this ticket now.
